docs/guide.rst

```diff
@@ -941,12 +941,12 @@ in event handlers in ``EventM``:
      mExtent <- Brick.Main.lookupExtent FooBox
      case mExtent of
        Nothing -> ...
-       Just (Extent _ upperLeft (width, height) offset) -> ...
+       Just (Extent _ upperLeft (width, height)) -> ...
```

Comment:
This field has been removed in https://github.com/jtdaugherty/brick/commit/249781a57b3c28ea50529a7f8d0eb3c8470dd433